### ISSUE TITLE
Different viewer handling in `Tex_ViewLaTeX()` and `Tex_ForwardSearchLaTeX()`

### ISSUE THREAD

**Issue by mowgli** [vim-latex/vim-latex#208]
I usually have only `g:Tex_ViewRule_pdf` set to my viewer (zathura). I often had the trouble, that when opened with `,lv` and then `,ls`, I get a second window while when I directly start with `,ls`, I only get one window.

I did debug that today and found, that both are using complete different approach to open the viewer. First is that `Tex_ViewLaTeX()` could be overwritten by `g:Tex_ViewRuleComplete_pdf` while `Tex_ForwardSearchLaTeX()` only respects `g:Tex_ViewRule_pdf`.

But the main problem lies in the fact that `Tex_ViewLaTeX()` uses `%:p:t:r`.pdf while `Tex_ForwardSearchLaTeX()` uses the equivalent of `%:p:r`.pdf on unix and `%:t:r`.pdf on windows.

Please, please, unify that a bit to have a accountable behaviour. Especially, use expanded or not expanded file name in both locations consistently.

**Comment by mowgli** [comment]
Ps. My leader is `,`, not `\`.

**Comment by gerw** [comment]
I agree that these two routines are some kind of mess... However, I do not understand your setup. What is
````
echo has('win32')
````
for you?

Does it help, if you use the full path in `Tex_ViewLaTeX`, i.e., `:t:r` on line 243?

**Comment by mowgli** [comment]
Ah, and `:t:r` is even worse as it is NOT the full path. `:p` is full path and `:t` is 'tail'. Here is the part from vim `builtin.txt`:
```
                Modifiers:
                        :p              expand to full path
                        :h              head (last path component removed)
                        :t              tail (last path component only)
                        :r              root (one extension removed)
                        :e              extension only
```

**Comment by gerw** [comment]
Yes, `:t:r` is not the full path -- maybe I meant `:p:r`.

What about using the `:p:r` in lines 240 and 243; followed by
````
let mainfname = shellescape(mainfname, 1)
````
to deal with possible spaces in the directory name?
